FreeRTOS/FreeRTOS
# Update SMP unit test for vTaskDelete change

Update SMP unit test for vTaskDelete change

Description
-----------
This PR is to update for the vTaskDelete implementation change in https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/958

Test Steps
-----------
Before this PR there will be failed test cases. Reference [link](https://github.com/FreeRTOS/FreeRTOS-Kernel/actions/runs/7581572277/job/20649454126?pr=958).

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
